project_national/fact_sheets_category_6.yml

```diff
@@ -0,0 +1,752 @@
+      - &fuel_oil_wallfloor_furnace_secondary_heating_type_fuel
+        option: HVAC Secondary Heating Type And Fuel|Fuel Oil Fuel Wall/Floor Furnace
+        apply_logic:
+          - HVAC Heating Type And Fuel|Fuel Oil Fuel Wall/Floor Furnace
+  # Set secondary fuel Wall/Floor Furnace efficiency levels
+      - &secondary_wallfloor_furnace_60
+        option: HVAC Secondary Heating Efficiency|Fuel Wall/Floor Furnace, 60% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Wall/Floor Furnace, 60% AFUE
+      - &secondary_wallfloor_furnace_68
+        option: HVAC Secondary Heating Efficiency|Fuel Wall/Floor Furnace, 68% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Wall/Floor Furnace, 68% AFUE
+
+#### ducted furnace ####
+  # - upgrade_name: Dual-Fuel Natural Gas Furnace 60% AFUE
+      - &ducted_ASHP_EnergyStar_exist_backup_shared_ducts_60_ng
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 16, 9.2 HSPF, Integrated Backup, 60% AFUE NG, 5F-40F switchover band
+        apply_logic: &dual-fuel_60pctAFUE_NG_furnace_apply_logic
+          - HVAC Has Ducts|Yes
+          - HVAC Heating Efficiency|Fuel Furnace, 60% AFUE
+          - HVAC Heating Type And Fuel|Natural Gas Fuel Furnace        
+        costs: *costs_ashp_ducted_SEER_16_9-2_HSPF
+        lifetime: *lifetime_ashp_ducted_SEER_16_9-2_HSPF
+      - &ducted_ASHP_EnergyStar_exist_backup_shared_ducts_76_ng
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 16, 9.2 HSPF, Integrated Backup, 76% AFUE NG, 5F-40F switchover band
+        apply_logic: &dual-fuel_76pctAFUE_NG_furnace_apply_logic
+          - HVAC Has Ducts|Yes
+          - HVAC Heating Efficiency|Fuel Furnace, 76% AFUE
+          - HVAC Heating Type And Fuel|Natural Gas Fuel Furnace        
+        costs: *costs_ashp_ducted_SEER_16_9-2_HSPF
+        lifetime: *lifetime_ashp_ducted_SEER_16_9-2_HSPF
+      - &ducted_ASHP_EnergyStar_exist_backup_shared_ducts_80_ng
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 16, 9.2 HSPF, Integrated Backup, 80% AFUE NG, 5F-40F switchover band
+        apply_logic: &dual-fuel_80pctAFUE_NG_furnace_apply_logic
+          - HVAC Has Ducts|Yes
+          - HVAC Heating Efficiency|Fuel Furnace, 80% AFUE
+          - HVAC Heating Type And Fuel|Natural Gas Fuel Furnace        
+        costs: *costs_ashp_ducted_SEER_16_9-2_HSPF
+        lifetime: *lifetime_ashp_ducted_SEER_16_9-2_HSPF
+      - &ducted_ASHP_EnergyStar_exist_backup_shared_ducts_92-5_ng
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 16, 9.2 HSPF, Integrated Backup, 92.5% AFUE NG, 5F-40F switchover band
+        apply_logic: &dual-fuel_92-5pctAFUE_NG_furnace_apply_logic
+          - HVAC Has Ducts|Yes
+          - HVAC Heating Efficiency|Fuel Furnace, 92.5% AFUE
+          - HVAC Heating Type And Fuel|Natural Gas Fuel Furnace        
+        costs: *costs_ashp_ducted_SEER_16_9-2_HSPF
+        lifetime: *lifetime_ashp_ducted_SEER_16_9-2_HSPF
```

Comment:
If we're including lifetimes in the yamls they should be addressed in some fashion in the intent document